Makefile

```diff
@@ -57,7 +60,7 @@ inspection_test:
 healthcheck_test:
 	make -C ./scripts/feature_tests/healthcheck/
 
-feature_tests: setup_env inspection_test remediation_test healthcheck_test cleanup_env pivoting_test node_reuse_test repivoting_test
+feature_tests: setup_env inspection_test remediation_test healthcheck_test boot_iso_test cleanup_env pivoting_test node_reuse_test repivoting_test
```

Comment:
Should we have the boot iso test in between, or should we have it right before re-pivoting? Because this test will remove one worker node if I understand right.

Reply:
Trying to figure that out.
I was testing by adding after `healthcheck_test`. Apparently, there are not provisioned nodes at that stage. 
Will try your suggestion. thanks.

Reply:
Added it just right before `repivoting_test`. 
Waiting for the result.